ci/Dockerfile-envoy

```diff
@@ -40,7 +40,8 @@ CMD ["envoy", "-c", "/etc/envoy/envoy.yaml"]
 
 
 # STAGE: envoy-distroless
-FROM gcr.io/distroless/base-debian11:nonroot AS envoy-distroless
+# gcr.io/distroless/base-debian11:nonroot
+FROM gcr.io/distroless/base-debian11@sha256:764b74b1789b4ec9044e6f20bb938d077fe8c7bcf9e4d3767eebb440c5d76f11 AS envoy-distroless
```

Comment:
Do we have a way (i.e. like dependabot) to automatically update this? What is the distroless sha for this if we pin?

Reply:
the hope is that dependabot will update this - config is here https://github.com/envoyproxy/envoy/blob/ac192f7e788b3240d608e6dd8793187715b1ef3b/.github/dependabot.yml#L49-L52

there have been some docker updates i believe - but not many - im guessing its because they arent currently pinned and therefore dont need updating